cmd/notary/keys.go

```diff
@@ -10,37 +11,81 @@ import (
 	"time"
 
 	"github.com/docker/notary/keystoremanager"
+	"github.com/docker/notary/pkg/passphrase"
 	"github.com/docker/notary/trustmanager"
 
 	"github.com/spf13/cobra"
 )
 
 func init() {
-	cmdKeys.AddCommand(cmdKeysRemoveRootKey)
-	cmdKeys.AddCommand(cmdKeysGenerateRootKey)
+	cmdKey.AddCommand(cmdKeyList)
+	cmdKey.AddCommand(cmdKeyRemoveRootKey)
+	cmdKey.AddCommand(cmdKeyGenerateRootKey)
+
+	cmdKeyExport.Flags().StringVarP(&keysExportGUN, "gun", "g", "", "Globally unique name to export keys for. A new password will be set for all the keys. Output format is a zip archive.")
```

Comment:
spewing binary garbage to the terminal makes it easy to `|` commands, if somebody has some desire to pipe it directly into an upload via curl or some other madness, so I think that behaviour is acceptable, possibly even desirable.

I feel a little icky about the absence of any specific instruction doing an export all, so I'd really prefer to see a `--all` flag.